script.js

```diff
@@ -44,5 +47,35 @@ function addTeamMember(event) { //eventlistener
     renderStats();
 }
 
+//below is light/dark toggle for the page, new consts will be at the top.
+
+function lastKnownMode () {
```

Comment:
I think we can just do one function and one class for 'dark', also this darkmode toggle function is being defined inside of lastKnownMode().